src/w3d/renderer/assetmgr.h

```diff
@@ -35,8 +35,8 @@ class W3DAssetManager
 {
 public:
     virtual ~W3DAssetManager();
-    virtual bool Load_3D_Assets(const char *filename);
     virtual bool Load_3D_Assets(FileClass &asset_file);
```

Comment:
is the order reversed when compiled?

Reply:
From my investigation in the asset manager pr this was back to front in thyme. The first virtual was dtor followed by FileClass and the third was const char*.